### PR TITLE
Add 'cache' support.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: package.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 name: cli
 description: Tools, like an argument parser, to create command-line applications.
-
 environment:
   sdk: ^2.0.0-alpha.100
+dependencies:
+  fs:
+    path: ../pkg-fs
```
Comment (kasperl) [inline package.yaml:7]: Why is this a local path?

File: tests/package.lock [hunk 2]
```diff
@@ -1,5 +1,19 @@
+sdk: ^2.0.0-alpha.64
 prefixes:
   cli: ..
+  host: pkg-host
 packages:
   ..:
     path: ..
+    prefixes:
```
Comment (kasperl) [inline tests/package.lock:8]: Why do we get host and fs through a local path?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tests/package.lock (hunk 2 above)
Comment (floitsch) [reply tests/package.lock:8]: The fs package wasn't published yet.

File: package.yaml (hunk 1 above)
Comment (floitsch) [reply package.yaml]: The 'fs' package wasn't published yet.

**Pull request merged**